deploy.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -e
+
+RED="\033[1;31m"
+GREEN="\033[1;32m"
+NOCOLOR="\033[0m"
+
+stack_name="$1"
+secret_prefix="$2"
+
+if [ -z "$stack_name" ]
+then
+echo -e "😱 ${RED}stack name expected as first argument, e.g. ${GREEN}./deploy common-lambdas-my-name${NOCOLOR}"
+exit 1
+fi
+
+if [ -z "$secret_prefix" ]
+then
+  secret_prefix="common-lambdas"
+fi
```

Comment:
☝️ note the default secret prefix of `common-lambdas` which will be set on secrets in dev

Reply:
At present the template.yaml file does not have a SecretPrefix parameter.
Therefore, I think it can be removed.
However, new parameters are required for `AuditEventNamePrefix` & `CriIdentifier`

Reply:
@tal-nagra I've updated the script to remove the secretprefix and add optional AuditEventNamePrefix & CriIdentifier args